src/components/procedure/_enable-turbo-mode.md

```diff
@@ -0,0 +1,9 @@
+:::tip Try Turbo mode (beta)
+Semgrep can automatically scan for findings using the Semgrep OSS Engine, on any change in the Rule or Sample Code panes.
```

Comment:
I was just checking [the pane definition](https://developers.google.com/style/ui-elements#term-windows) also in [Stack Exchange](https://ux.stackexchange.com/questions/124588/whats-the-difference-between-a-panel-and-pane). Looks correct - as pane is a section of a window. So the Editor is a page, with a window that takes roughly four-fifths of its size, with a left sample code pane and a right test code pane. Feel free to resolve this comment, and if you want send me more details so I can be sure we shouldn't use a "panel". :D